### DISCUSSION TITLE
Round screens

### DISCUSSION THREAD

**Discussion by SergeDubovsky** [KlipperScreen/KlipperScreen#1239] [Ideas]
Knomi is an obvious candidate, but we also have a ton of [interesting hardware](https://www.waveshare.com/5inch-1080x1080-LCD.htm) available. Any ideas how to make it round?

**Comment by alfrix** [comment]
klipperscreen is not designed for round screens, an entirely different UI design would be needed to have a good experience, this will not happen on this repo, it might work as is, but no particular support will be added
